editor/src/components/navigator/navigator-item/component-picker-context-menu.tsx

```diff
@@ -342,21 +342,7 @@ function moreItem(
   showComponentPickerContextMenu: ShowComponentPickerContextMenu,
 ): ContextMenuItem<unknown> {
   return {
-    name: (
-      <FlexRow>
-        <div
-          style={{
-            width: 18,
-            height: 18,
-            display: 'flex',
-            justifyItems: 'center',
-            alignItems: 'center',
-            position: 'relative',
-          }}
-        ></div>{' '}
-        More...
-      </FlexRow>
-    ),
+    name: <FlexRow style={{ paddingLeft: 22 }}>More...</FlexRow>,
```

Comment:
```suggestion
    name: <FlexRow style={{ paddingLeft: 22 }}>More…</FlexRow>,
```

(I know this is not a change introduced here, but while we might as well)